packages/relay/src/lib/eth.ts

```diff
@@ -1746,6 +1748,52 @@ export class EthImpl implements Eth {
     return numberTo0x(block.count);
   }
 
+  private async validateBlockHashAndAddTimestampToParams(params: any, blockHash: string, requestIdPrefix?: string) {
```

Comment:
Q: why this re-addition of methods moved out from previous PRs?
This might be a bad rebase.

Reply:
Yes, it was a bad rebase.